### PR TITLE
rename .template.yml to .yml.template

### PR DESCRIPTION
**Pull request by kevinAlbs**
The .template.yml files interfered with the Makefile that converted yml to json.

### CODE REVIEW

File: source/client-side-encryption/tests/generate.py [hunk 1]
```diff
@@ -25,11 +25,11 @@
 filepath = sys.argv[1]
 filedir = os.path.dirname(filepath)
 (filename, ext) = os.path.splitext(os.path.basename(filepath))
-if ext != ".yml":
-    print("Input file must end with .yml")
+if ext != ".template":
+    print("Input file must end with .template.yml")
```
Comment (jmikola) [inline source/client-side-encryption/tests/generate.py]: Should this be `.yml.template`?

File: source/client-side-encryption/tests/local.yml [hunk 2]
```diff
@@ -1,56 +0,0 @@
-runOn:
```
Comment (kevinAlbs) [inline source/client-side-encryption/tests/local.yml]: This slipped in originally, but ended up being renamed and hence no longer used.

File: source/client-side-encryption/tests/generate.py [hunk 3]
```diff
@@ -25,11 +25,11 @@
...
     sys.exit(1)
 (filename, ext) = os.path.splitext(filename)
-if ext != ".template":
+if ext != ".yml":
```
Comment (jmikola) [inline source/client-side-encryption/tests/generate.py]: I'm not sure how this check differs from the previous check, but is this still accurate?

Is this attempting to validate the output filename (derived from the original template filename)? Perhaps it's just the error message below that's inaccurate.

Comment (kevinAlbs) [reply source/client-side-encryption/tests/generate.py]: Yeah the error message needed to be updated.

File: source/client-side-encryption/tests/generate.py (hunk 1 above)
Comment (kevinAlbs) [reply source/client-side-encryption/tests/generate.py]: Fixed.

File: source/client-side-encryption/tests/generate.py (hunk 3 above)
Comment (jmikola) [reply source/client-side-encryption/tests/generate.py]: Ah, I see what the logic is now. You're checking the order of extensions. :+1:

**Pull request merged**